Sources/Gherkin/Models.swift

```diff
@@ -84,11 +94,13 @@ public enum Scenario {
 
 /// A gherkin Scenario: that does not have examples.
 public struct ScenarioSimple: Codable {
+    public var tags: [Tag]
     public var name: String
     public var textDescription: String?
     public var steps: [Step]
 
-    public init(name: String, description: String?, steps: [Step]) {
+    public init(tags: [Tag], name: String, description: String?, steps: [Step]) {
```

Comment:
Adding tags to the front of the array would be a breaking change so let’s add it to the end with a default value of nil.